Entities/MOSRotating.cpp

```diff
@@ -324,7 +337,11 @@ int MOSRotating::ReadProperty(const std::string_view &propName, Reader &reader)
         Attachable *pAttachable = new Attachable;
         reader >> pAttachable;
         m_Attachables.push_back(pAttachable);
-    }
+	} else if (propName == "SpecialBehaviour_AddWound") {
```

Comment:
Any reason this needs to be a special behaviour? Might be useful for modders.

Reply:
Yeah, I thought about this for a while and ended up deciding it was special, for the following reasons:

1. There was no old behaviour to add wounds via INI, so it obviously wasn't something people wanted much. If someone wants to add wounds on create, Lua has them covered anyway, so this doesn't keep the ability from them.

2. If wounds were allowed to be added by INI normally, I'd have to set the clear flag for situations with wounds as well, which would make there be even more copying reference data, then deleting it and reading from INI instead. Obviously it's not the end of the world, but I figured I'd skip that extra bit of work since, as mentioned in point 1, it's not even an existing feature.

Reply:
Fair enough. Sounds good to me.